src/main/resources/CU-ApplicationResources.properties

```diff
@@ -500,7 +500,7 @@ validation.error.concur.sae.group.with.orphaned.cash.advance=Line was not proces
 
 validation.error.concur.request.extract.cash.advance.invalid.unique.identifier=Unique identifier Cash Advance Key for Request Extract generated cash advance request was detected as being NULL or blank.
 validation.error.concur.sae.not.requested.cash.advance=Standard Accounting Extract Detail Line is not a Requested Cash Advance.
-validation.error.concur.sae.not.cash.admin.approved=SAE cash advance request not approved. Detected Cash Advance Admin Indicator of {0}.
+validation.error.concur.sae.not.cash.admin.approved=SAE cash advance request not approved or applied. Detected Cash Advance Admin Indicator of {0}.
```

Comment:
Since you're expanding this error message to include the not-applied case, should the message's key be renamed to reflect that?